app/controllers/users/profiles_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Users::ProfilesController < ApplicationController
+  def show
+    @profile = Profile.find(params[:id])
```

Comment:
Can we access one of these through the other? Either profile.user or user.profile?